frontend/src/v4/routes/components/highlight/highlight.styles.ts

```diff
@@ -15,11 +15,12 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+import { isV5 } from '@/v4/helpers/isV5';
 import styled from 'styled-components';
 import { FONT_WEIGHT } from '../../../styles';
 
 export const Mark = styled.mark`
-	color: #06563c;
+	color: ${isV5() ? ({ theme }) => theme.palette.primary.main : '#06563c'};
```

Comment:
Is too complicated to have this in v4 overrides?

Reply:
I tried this. The problem is that this component is being rendered outside of the V4Overrides DOM (This is for the highlight that you see when typing a users name into the add user popover )
![image](https://user-images.githubusercontent.com/62481250/207573220-c1c89503-fa2e-4b94-a586-404417400ad1.png)
So this seemed like the easiest solution